### PR TITLE
Update generation script to match API changes

### PR DESCRIPTION
**Pull request by nathanaelhoun**
## Changes

- [x] Update HTTP responses codes checking to match API changes introduced in #247.

### CODE REVIEW

**Review by FrancoisPog — APPROVED** [review]
LGTM !

File: server/db/fill-fake-data.js [hunk 1]
```diff
@@ -166,6 +166,12 @@ function createFakeDuelWithAPI(config, usernames, MAX_ANSWER, index) {
 
       try {
         const [user1, user2] = faker.random.arrayElements(usernames, 2);
+        if (user1 === user2) {
+          console.warn(`     ${user1} can't defy himself/herself!`);
```
Comment (valentinperignon) [inline server/db/fill-fake-data.js:170]: him/her great great great... And what is the person is a non-binary person?

**Review by valentinperignon — APPROVED** [review]
LGTM

File: server/db/fill-fake-data.js (hunk 1 above)
Comment (valentinperignon) [reply server/db/fill-fake-data.js:170]: Should be `himself/herself/himherself/herhimself/itself/whateverelse`

Comment (nathanaelhoun) [reply server/db/fill-fake-data.js:170]: If prefer `((him)?(her)?self|themselves)`

**Pull request merged**